### PR TITLE
Turn on SiP for 1990e

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4478

Pretty much what the title says. No prefill, that requires backend work.

screen shot 2017-09-05 at 1 00 59 pm

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM.

File: test/e2e/edu-helpers.js [hunk 1]
```diff
@@ -15,6 +18,56 @@ function initApplicationSubmitMock(form) {
       }
     }
   });
+  /* eslint-disable camelcase */
+  mock(token, {
```
Comment (cvalarida) [inline test/e2e/edu-helpers.js]: What's all this for?

**Review by cvalarida — APPROVED** [review]
Looks good, I'm just curious what all that `mock()`ing is for. _[Insert pun here]_

File: test/e2e/edu-helpers.js (hunk 1 above)
Comment (jbalboni) [reply test/e2e/edu-helpers.js]: When I ran the test locally, the buttons weren't showing up, which appeared to be because the login requests all failed.

**Pull request merged**